configuration/analyzer_config.json

```diff
@@ -772,6 +772,21 @@
       }
     }
   },
+  "Filescan": {
+    "type": "file",
+    "python_module": "filescan.FileScan",
+    "description": "Check Filescan",
+    "disabled": false,
+    "external_service": true,
+    "leaks_info": false,
+    "config": {
+      "soft_time_limit": 180,
+      "queue": "default"
```

Comment:
queue -> `long`.

Reply:
Yes i think this was the issue, can u explain the reason behind changing it to  `long` so that i may be aware of it in my upcoming contributions

Reply:
https://intelowl.readthedocs.io/en/latest/Advanced-Usage.html#multi-queue